### PR TITLE
Added sprinkles of OSE corporate design

### PR DESCRIPTION
**Pull request by rafaelsuchy**
Steps performed 
- [x] Added logo on title page (down right) (currently outcommented)
- [x] Added logo in author field (preferred solution)
- [x] Changed hyperref color to OSEBlue (and added OSEBlueLight)
- [x] Changed font to OSE font (and added Preamble)
- [x] Compilation check passed

### CODE REVIEW

**Comment by peisenha** [comment]
Looks awesome, but can we create a black-white version of our logos and use that intead? I always try to keep all bw in the papers and then only use color in the presentations. It is sometimes hard to predict how colored stuff looks like when people print it. Let me know ...

File: handout/sections/title.tex [hunk 1]
```diff
@@ -2,5 +2,8 @@
 
 \title{Eckstein-Keane-Wolpin models}
 \subtitle{An invitation for transdisciplinary collaboration}
-\author{OSE@Bonn\thanks{Corresponding author: Philipp Eisenhauer, peisenha@uni-bonn.de. We thank  Jano\'s Gabler, Annica Gehlen, Boryana Ilieva, Fedor Ishakov, Lena Janys, Moritz Mendel, Tim Mensinger, Tobias Raabe, Klara R\"ohl, and John Rust for their valuable input. Luis Wardenbach provided outstanding research assistance.}}
+\author{
+	% \includegraphics[height=2.5cm]{../material/crop-ose-logo.pdf} \vspace{-1.8cm} \\ \hspace{6.1cm}
```
Comment (peisenha) [inline handout/sections/title.tex]: Good to keep both logos in the material folder, but please deleted commented out line.

File: handout/sections/header.tex [hunk 2]
```diff
@@ -128,6 +135,9 @@
   {\par\normalfont\hfill--\ \chapquote@author\hspace*{\@tempdima}\par\bigskip}
 \makeatother
 
+% OSE font settings
```
Comment (peisenha) [inline handout/sections/header.tex]: Can we get rid fo this? and the other file?

File: handout/sections/titlepage.tex [hunk 3]
```diff
@@ -15,6 +15,17 @@
 \noindent We present background material on a class of structural microeconometric models to facilitate transdisciplinary collaboration in their future development. We describe the economic framework, mathematical formulation, and calibration procedures for so-called Eckstein-Keane-Wolpin (EKW) models. We specify, simulate, and calibrate an example using our group's research codes \verb+respy+ and \verb+estimagic+. We summarize our efforts drawing on research outside economics to address their computational challenges and improve their reliability and interpretability.
 \end{abstract}
 
+% Comment out for right down logo
```
Comment (peisenha) [inline handout/sections/titlepage.tex]: Please remove.

Comment (rafaelsuchy) [reply handout/sections/titlepage.tex]: removed

File: handout/sections/header.tex (hunk 2 above)
Comment (rafaelsuchy) [reply handout/sections/header.tex]: Removed the extra preamble and any font-settings that require .sty files.

File: handout/sections/title.tex (hunk 1 above)
Comment (rafaelsuchy) [reply handout/sections/title.tex]: Only black-white logo included in paper.

**Comment by peisenha** [comment]
B-E-A-U-T-I-F-U-L. Thanks!

**Pull request merged**